codes/classical/bits/nordstrom_robinson.yml

```diff
@@ -0,0 +1,42 @@
+
+code_id: nordstrom_robinson
+physical: bits
+logical: bits
+
+name: 'Nordstrom-Robinson Code'
+introduced: '\cite{doi:10.1016/S0019-9958(67)90835-2}'
+
+description: 
+    - 'Nordstrom-Robinson code is a non-linear binary code of length \(n=16\), size \(N = 256\), and minimum distance \(d = 6\). This was first discovered idependently by A.W. Nordstrom, and J.P. Robinson \cite{doi:10.1016/S0019-9958(67)90835-2} in 1967 and N.V. Semakov, and V.A. Zinov'ev in 1969.'
+    - 'This code can be constructed using extended Golay code of length 24 by first selecting a set of Golay codewords satisfying certain conditions and then deleteing specific coordinates.' 
+    - 'While the Nordstrom-Robinson code itself is non-linear, it can be constructed by applying Gray-mapping to a certain linear octacode over \(\mathbb{Z_4}\).'
+    - 'Nordstrom-Robinson code is the first Kerdock code as well as the first Preparata code. This implies that it is the dual code of itself in the sense that the MacWilliams transform of its distance distribution is equal to itself.'
+    - 'It is an orthogonal array of strength 5.'
+    - 'It has been proven that linear code of length \(n=16\), size \(N = 256\), and minimum distance \(d = 6\)cannot exist. More precisely, for all linear codes with \(n = 16\) and \(d = 6\), \(N\) should be strictly less than 256.' 
+ 
+protection: 'This code can correct upto 2 bit flips or 5 erasures.' 
+     
+features:
+
+  rate: '= 1/2'
+  
+# cite following
+# Introduced-- An optimum nonlinear code -- doi:10.1016/S0019-9958(67)90835-2,,,,, N. V. Semakov, V. A. Zinov'ev, “Complete and Quasi-complete Balanced Codes”, Probl. Peredachi Inf., 5:2 (1969), 14–18; Problems Inform. Transmission, 5:2 (1969), 11–13
+
+# Construction -1 The theory of error correcting codes - chapter 11.
+# Construction -2 The Nordstrom-Robinson Code is the Binary Image of the Octacode
+# Orthogonal array specification -- http://mint.sbg.ac.at/desc_CNordstromRobinson.html#bib-CNordstromRobinson-3
+# 
+
+relations:
+  parents:
+    - code_id: kerdock
```

Comment:
Another student submitted kerdock and had this for nordstrom; please integrate this informaiton into a detail entry for kerdock: 'Specializes Kerdock code for \(m = 4\). Consequently, the Kerdock set is a set of 8 matrices in \(\mathbb(F)_{2}^{4X4}\). The resulting code is binary with parameters [16, 256, 6].'

Reply:
Add a detail for preparata as well